<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="fastBake_checkBox">
     <property name="text">
      <string>Fast bake</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="smartBake_checkBox">
     <property name="text">
      <string>Smart</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sample By :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="Sample_SpinBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="toAnother_checkBox">
     <property name="text">
      <string>to another</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="selectedObj_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="targetObj_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="C2L_pushButton">
       <property name="text">
        <string>CON &gt; LOC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="L2C_pushButton">
       <property name="text">
        <string>LOC &gt; CON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Close_pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
